module-types/Appear/ECx110.yaml

```diff
@@ -0,0 +1,41 @@
+---
+manufacturer: Appear
```

Comment:
Previously we have named this manufacturer `AppearTV`.
The modules should match.

We must either change the name of these modules to `AppearTV`, or the name of the manufacturer of the devices to `Appear`.

Reply:
I know, but we should change to only "Appear". I believe the company has been rebranded...